Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuth.h

```diff
@@ -140,6 +140,13 @@
 ///
 - (BOOL)clearStoredAccessTokens;
 
+///
+/// When NO, the Dropbox web login page will show a sign up link below the
```

Comment:
The sign up flow doesn't _require_ a paid account, but I believe without `disable_signup=true` it is possible to navigate to an upgrade page which would still be against Apple's rules.

That being the case, if you want this for use outside of the App Store somehow, that's fine (as long as the documentation is clarified) but otherwise I think you'll still need `disable_signup=true` so there wouldn't be any use in merging this.